newIDE/app/src/ObjectsList/index.js

```diff
@@ -296,15 +296,14 @@ export default class ObjectsList extends React.Component<Props, State> {
   _rename = (objectWithContext: ObjectWithContext, newName: string) => {
     const { object } = objectWithContext;
 
-    if (getObjectWithContextName(objectWithContext) === newName) {
-      return;
-    }
-
     this.setState({
       renamedObjectWithContext: null,
     });
 
-    if (this.props.canRenameObject(newName)) {
+    if (
+      getObjectWithContextName(objectWithContext) === newName ||
```

Comment:
See my comment in the other PR, the condition seems like it should be the reverse.

Reply:
I have removed the commit from this PR